args/bosses.py

```diff
@@ -11,6 +11,8 @@ def parse(parser):
                         help = "Boss battles randomized")
     bosses.add_argument("-bmbd", "--mix-bosses-dragons", action = "store_true",
                         help = "Shuffle/randomize bosses and dragons together")
+    bosses.add_argument("-bmbs", "--mix-bosses-statues", action = "store_true", default = True,
```

Comment:
defaults to true to retain backwards compatibility with flag strings that want to retain statues in boss pool